app/models/knight.rb

```diff
@@ -1,2 +1,19 @@
 class Knight < Piece
+  def obstructed_diagonally?
```

Comment:
Just remembered that my PR #28 included an edit to *rubocop.yml*: on line #28 I changed `120` to `130`. Does it matter that I didn't mention it when I did `git commit --am ...`?